service/build.gradle

```diff
@@ -144,6 +148,7 @@ sonarqube {
     property "sonar.pitest.mode", "reuseReport"
     property "sonar.pitest.reportsDirectory", "build/reports/pitest"
     property "sonar.coverage.exclusions", "**/model/**"
+    property "sonar.exclusions", "**/model/**"
```

Comment:
it would be nicer to have this just target the two files rather than all...

Reply:
I could of sworn I fixed this to include the above array variable. Let me try again...